packages/bpk-component-barchart/hocs.js

```diff
@@ -16,13 +16,18 @@
  * limitations under the License.
  */
 
+/* @flow strict */
+
 import isEqual from 'lodash/isEqual';
-import React, { Component } from 'react';
+import React, { Component, type ComponentType } from 'react';
 import { wrapDisplayName } from 'bpk-react-utils';
 
 // eslint-disable-next-line import/prefer-default-export
-export const withSelectedState = ComposedComponent => {
-  class WithSelectedState extends Component {
+export const withSelectedState = (ComposedComponent: ComponentType<any>) => {
+  type State = {
+    selectedPoint: ?any,
```

Comment:
We typically try to avoid `any` unless absolutely necessary, is there something more specific we can use here?